litellm/proxy/pass_through_endpoints/llm_passthrough_endpoints.py

```diff
@@ -259,6 +259,10 @@ async def bedrock_proxy_route(
     if not encoded_endpoint.startswith("/"):
         encoded_endpoint = "/" + encoded_endpoint
 
+    # Remove agent from knowledgebase url /agents/knowledgebases/ -> /knowledgebases/
+    if "knowledgebases" in encoded_endpoint:
```

Comment:
hey @LBirk this is great. Love the idea of using `agents/` in the pass through. 

Can you add a unit test to make sure this works as expected + no future regressions. 

This would be a good place for it - https://github.com/BerriAI/litellm/blob/0dbf71291e9ed261a15ada370668c3a6598ed990/tests/local_testing/test_pass_through_endpoints.py#L48